### PR TITLE
[libass] fix libdir

### PR DESCRIPTION
**Pull request by lovedheart**
#1981

### CODE REVIEW

**Comment by uberhacker** [comment]
There is an update available for this library.  See https://github.com/libass/libass/releases/download/0.14.0/libass-0.14.0.tar.xz.

File: packages/libass.rb [hunk 1]
```diff
@@ -3,22 +3,11 @@
...
-  })
-  binary_sha256 ({
-    aarch64: '0b0996d9626e827c7a22b95c7eb59217b657afa2044934884167deaae217d106',
-     armv7l: '0b0996d9626e827c7a22b95c7eb59217b657afa2044934884167deaae217d106',
-       i686: 'd9ed5ea138dda3cff16e20b33534cf3cd7f3cead250cd7c5b74347bf517938fa',
-     x86_64: 'dbc621f3d5fb1d39338fbd2b7e7f1237b12d67f8e46f95a999550680ab0ecbfd',
-  })
 
   depends_on 'automake' => :build
   depends_on 'autoconf' => :build
```
Comment (uberhacker) [inline packages/libass.rb]: You can remove all the build dependencies since they're already part of Chromebrew "core".

File: packages/libass.rb [hunk 2]
```diff
@@ -3,34 +3,15 @@
...
-     armv7l: '0b0996d9626e827c7a22b95c7eb59217b657afa2044934884167deaae217d106',
-       i686: 'd9ed5ea138dda3cff16e20b33534cf3cd7f3cead250cd7c5b74347bf517938fa',
-     x86_64: 'dbc621f3d5fb1d39338fbd2b7e7f1237b12d67f8e46f95a999550680ab0ecbfd',
-  })
-
-  depends_on 'automake' => :build
-  depends_on 'autoconf' => :build
-  depends_on 'freetype'
-  depends_on 'fribidi'
-  depends_on 'fontconfig'
```
Comment (uberhacker) [inline packages/libass.rb]: We probably want to leave the non-build dependencies, correct?

Comment (lovedheart) [reply packages/libass.rb]: I would like uncomment it. I think it's better.

Comment (lovedheart) [reply packages/libass.rb]: I think we don't change any dependency-related config.

File: packages/libass.rb [hunk 3]
```diff
@@ -3,34 +3,21 @@
...
-     x86_64: 'dbc621f3d5fb1d39338fbd2b7e7f1237b12d67f8e46f95a999550680ab0ecbfd',
-  })
+  version '0.14.0'
+  source_url 'https://github.com/libass/libass/releases/download/0.14.0/libass-0.14.0.tar.xz'
+  source_sha256 '881f2382af48aead75b7a0e02e65d88c5ebd369fe46bc77d9270a94aa8fd38a2'
 
   depends_on 'automake' => :build
   depends_on 'autoconf' => :build
+  depends_on 'libtool' => :build
   depends_on 'freetype'
```
Comment (uberhacker) [inline packages/libass.rb]: `freetype` is already a dependency of `fontconfig` so it can excluded.

File: packages/libass.rb [hunk 4]
```diff
@@ -3,34 +3,21 @@
...
-       i686: 'd9ed5ea138dda3cff16e20b33534cf3cd7f3cead250cd7c5b74347bf517938fa',
-     x86_64: 'dbc621f3d5fb1d39338fbd2b7e7f1237b12d67f8e46f95a999550680ab0ecbfd',
-  })
+  version '0.14.0'
+  source_url 'https://github.com/libass/libass/releases/download/0.14.0/libass-0.14.0.tar.xz'
+  source_sha256 '881f2382af48aead75b7a0e02e65d88c5ebd369fe46bc77d9270a94aa8fd38a2'
 
   depends_on 'automake' => :build
   depends_on 'autoconf' => :build
+  depends_on 'libtool' => :build
```
Comment (uberhacker) [inline packages/libass.rb]: We can remove all the `:build` dependencies, correct?

File: packages/libass.rb [hunk 5]
```diff
@@ -3,34 +3,20 @@
...
-       i686: 'd9ed5ea138dda3cff16e20b33534cf3cd7f3cead250cd7c5b74347bf517938fa',
-     x86_64: 'dbc621f3d5fb1d39338fbd2b7e7f1237b12d67f8e46f95a999550680ab0ecbfd',
-  })
-
-  depends_on 'automake' => :build
-  depends_on 'autoconf' => :build
-  depends_on 'freetype'
+  depends_on 'automake'
+  depends_on 'autoconf'
+  depends_on 'libtool'
```
Comment (uberhacker) [inline packages/libass.rb]: No, I mean these lines can be removed completely since they're already part of Chromebrew "core".  Sorry for the confusion. :)

**Pull request merged**